.github/workflows/ztest.yml

```diff
@@ -0,0 +1,69 @@
+name: ztest
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
```

Comment:
You might want to specify the ubuntu version, to prevent any future changes to effect test consistency.

Reply:
It's intentional, I prefer to use latest ubuntu (newer kernel too) and get problems ASAP.

Reply:
ubuntu-latest, isn;t the latest ubuntu release... it's LTS: Ubuntu 18.04

Reply:
Yep, you got me :) latest LTS on github CI, it will be 20.4 pretty soon.